tests/tests.json

```diff
@@ -1,12 +1,12 @@
 {
     "tests":
         [
-            {
+            /*{
                 "test name": "Argument, option and arguments map correctly",
                 "file name": "testBasics",
                 "args": "1 2.0 '?' -i:10 false one two three --testing",
                 "expect": 0
-            },
+            },*/
```

Comment:
Let's remove the comments to keep testing everything.

Reply:
😊 oops, forgot to uncomment